.github/workflows/build.yml

```diff
@@ -49,13 +49,17 @@ jobs:
             for: linux
             prepare: "debian-based"
             build_on: linux
-            use_image: ghcr.io/owtech/foundationdb-build:7.4.0-1.ow.build
-            parallel: 2
+            parallel: 5
+            image: foundationdb-build:7.4.0-5.ow.build
 
     runs-on: ${{ matrix.run_on }}
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      
+      - name: Set building repo
+        run: | 
+          echo "use_image=ghcr.io/${GITHUB_REPOSITORY_OWNER@L}/${{matrix.image}}" >> "$GITHUB_ENV"
```

Comment:
Usually personal forks do not have build images. So getting one from ghcr.io/owtech/foundationdb-build is a better approach.

Please revert it to the old state.